tools/test_utils.py

```diff
@@ -225,8 +228,7 @@ def run_tensorflow_pytests_from_artifacts(backend, ngraph_tf_src_dir,
     print("CURRENT DIR: " + os.getcwd())
 
     print("Patching TensorFlow using: %s" % patch_file)
-    result = call(["patch", "-p1", "-N", "-i", patch_file])
-    print("Patch result: %d" % result)
+    apply_patch(patch_file)
```

Comment:
@avijit-nervana please note, we replaced `call` to apply patch with this newly minted function `apply_patch` that allows only 2 cases... patch application successful, or patch was already applied, so skipped. In other cases it throws an error. This gives us some safety in detecting if the patch and the file/repo ever becomes out of sync